client/src/app/components/stardust/statistics/RichestAddresses.scss

```diff
@@ -0,0 +1,69 @@
+@import "../../../../scss/fonts";
+@import "../../../../scss/mixins";
+@import "../../../../scss/media-queries";
+@import "../../../../scss/variables";
+
+.richest-addr-section {
+    ul.richest-addr__list {
+        padding: 1rem;
+
+        li.richest-addr__entry {
+            @include font-size(14px);
+            font-family: $ibm-plex-mono;
+            font-weight: normal;
+            letter-spacing: 0.02em;
+            line-height: 26px;
+
+            @include tablet-down {
+                // flex-direction: column;
+                justify-content: space-between;
+            }
+
+            .entry__left {
+                flex-grow: 1;
+
+                .entry__idx {
+                    text-align: center;
+                    width:16px;
+                    margin-right: 1rem;
+                    color: var(--link-color);
```

Comment:
Maybe we should remove the link color so the number is distinguishable from the address. wdyt?